rm-main/src/ui/tabs/torrents/tasks/move_torrent.rs

```diff
@@ -0,0 +1,62 @@
+use crossterm::event::{KeyCode, KeyEvent};
+use ratatui::prelude::*;
+use transmission_rpc::types::Id;
+
+use crate::{
+    action::Action,
+    app,
+    transmission::TorrentAction,
+    ui::{components::Component, tabs::torrents::input_manager::InputManager, to_input_request},
+};
+
+pub struct MoveBar {
+    torrents_to_move: Vec<Id>,
+    ctx: app::Ctx,
+    input_mgr: InputManager,
+}
+
+impl MoveBar {
+    pub fn new(ctx: app::Ctx, to_move: Vec<Id>) -> Self {
+        let prompt = format!("New directory: ");
+
+        Self {
+            torrents_to_move: to_move,
+            input_mgr: InputManager::new(ctx.clone(), prompt),
```

Comment:
use InputManager::new_with_value (or something like this), with the value of the torrent's old directory